### ISSUE TITLE
Crash loading empty tox file

### ISSUE THREAD

**Issue by jetXS** [TouchDesigner/TouchEngine-UE4#86]
![crash_loading_empty_tox_file](https://user-images.githubusercontent.com/10253645/129733203-f60fb5e8-bbc7-4d20-aff3-ffba6781265c.png)

Was able to reproduce consistently.

1. Create Actor blueprint
2. Add a new TouchEngine Base component using Add Component menu in blueprint editor
3. Place attached .tox file (which is empty) in the TDToxFiles folder
4. Type in Component properties of BP the path tho that .tox file
5. It should crash

**Comment by jetXS** [comment]
[Test.tox.zip](https://github.com/TouchDesigner/TouchEngine-UE4/files/7000274/Test.tox.zip)

**Comment by jetXS** [comment]
I think it is due to the .tox being created in a version of TouchDesigner that is not on par with the lib.

Not a priority since it shouldn't occur for the average user but exceptions should be caught.

**Comment by jetXS** [comment]
Seems to be the same as #90 actually, just triggered differently.

**Comment by jetXS** [comment]
Blocked by #98

**Comment by jetXS** [comment]
Tested and confirmed as fixed.
